Integrations/ArcSightESMv2/ArcSightESMv2.py

```diff
@@ -345,6 +346,9 @@ def fetch():
     already_fetched = last_run.get('already_fetched', [])
 
     fields, query_results = get_query_viewer_results(events_query_viewer_id or cases_query_viewer_id)
+    # sort query_results by creation time
+    query_results.sort(key=lambda k: int(k.get('Start Time') or k.get('Create Time')))
```

Comment:
this could cause unexpected behavior
they should get ingested in the same order they appear in ArcSight

Reply:
@omercnet @anara123 the order is not always ascending (the opposite actually, most the times). since we now fetching in batches, we might lose the oldest cases/events. e.g., a query with for all the cases in the last 2 hours (and let's say there are 200)- if we'll use ArcSight order we will fetch the most recent first, wait for 15 minutes to the next fetch (default) and then get the next batch. this will take us at total an hour. In the meantime, the query viewer could update and discard the cases/events from 3 hours ago.

Reply:
not sure what you refer to but we wait 1 minute between fetches, not 15

so maybe consider reversing the order, not sorting?

Reply:
to add to what I said, the data from ArcSight is not always ordered, and even if it is, not always in ascending order.

Reply:
so let's please keep the same (dis)order so it's ingested in the same order as in the UI